misc/messaging.talon

```diff
@@ -0,0 +1,17 @@
+tag: user.messaging
+-
+# Navigation
+previous (workspace | server): user.messaging_workspace_previous()
+next (workspace | server): user.messaging_workspace_next()
+channel: user.messaging_open_channel_picker()
+channel <user.text>:
+    user.messaging_open_channel_picker()
+    insert(user.formatted_text(user.text, "ALL_LOWERCASE"))
+channel up: user.messaging_channel_previous()
+channel down: user.messaging_channel_next()
+([channel] unread last | gopreev): user.messaging_unread_previous()
+([channel] unread next | goneck): user.messaging_unread_next()
```

Comment:
I'm not sure if we have some sort of standard already in other abstract command sets for go previous and go next, but I don't think we use either of these. It may be worth considering.

Reply:
You mean like `([channel] unread next | go next | goneck)`? I'm fine with that. I was starting with the existing slack command wording wherever I could, but certainly happy to change it up if y'all have a clearer vision for the direction to take it.